pkg/fqdn/cache_test.go

```diff
@@ -691,9 +692,67 @@ func (ds *DNSCacheTestSuite) TestZombiesGC(c *C) {
 	})
 }
 
+func (ds *DNSCacheTestSuite) TestZombiesGCDeferredDeletes(c *C) {
+	now := time.Now()
+	zombies := NewDNSZombieMappings(defaults.ToFQDNsMaxDeferredConnectionDeletes)
+
+	zombies.Upsert(now.Add(0*time.Second), "1.1.1.1", "test.com")
+	zombies.Upsert(now.Add(1*time.Second), "2.2.2.2", "somethingelse.com")
+	zombies.Upsert(now.Add(2*time.Second), "3.3.3.3", "onemorething.com")
```

Comment:
wow, if you wanted to actually direct this domain to `3.3.3.3` then you could do so at the low low price of 4 million euros. What a steal!

Reply:
3 million is the most I'd pay. Otherwise how would I keep the 3s going (3,333,333.33 in fact)